target/ppc/translate/storage-ctrl-impl.c.inc

```diff
@@ -21,6 +21,20 @@
  * Store Control Instructions
  */
 
+static bool trans_SLBIE(DisasContext *ctx, arg_SLBIE *a)
+{
+    REQUIRE_64BIT(ctx);
+    REQUIRE_INSNS_FLAGS(ctx, SLBI);
+    REQUIRE_SV(ctx);
+
+#if !defined(CONFIG_USER_ONLY) && defined(TARGET_PPC64)
+    gen_helper_slbie(cpu_env, cpu_gpr[a->rb]);
```

Comment:
Helpers of insns that we move to decodetree should have the insn name uppercase:
```suggestion
    gen_helper_SLBIE(cpu_env, cpu_gpr[a->rb]);
```
And you'll need to change in helper.h and mmu-hash64.c too

Reply:
Fixed in 84bea988bde0c9816134d5657fe64ac1b96e538a